music_assistant/server/providers/tidal/__init__.py

```diff
@@ -526,13 +530,15 @@ async def _parse_album(self, album_obj: TidalAlbum, full_details: bool = False)
 
         album.upc = album_obj.universal_product_number
         album.year = int(album_obj.year)
+        available = album_obj.available
```

Comment:
you are 100% sure that an unavailable item also can not be retrieved anymore ? Because usually this indicates stream availability and you can still get the details. So what I'm trying to say is Unavailable != NotFound